libs/common/src/key-management/crypto/services/encrypt.service.implementation.ts

```diff
@@ -31,7 +39,9 @@ export class EncryptServiceImplementation implements EncryptService {
 
   // Handle updating private properties to turn on/off feature flags.
   onServerConfigChange(newConfig: ServerConfig): void {
-    return;
+    const old = this.useSDKForDecryption;
+    this.useSDKForDecryption = getFeatureFlagValue(newConfig, FeatureFlag.UseSDKForDecryption);
+    this.logService.debug("updated sdk decryption flag", old, this.useSDKForDecryption);
```

Comment:
We've recently started tagging the logs so we know where they are coming from and can filter easily (this goes for the other logs too):

```suggestion
    this.logService.debug("[EncryptService] Updated sdk decryption flag", old, this.useSDKForDecryption);
```